src/redux/reducers/cacheReducer.ts

```
@@ -229,6 +233,39 @@ const cacheReducer = (state = initialState, action) => {
       return { ...state };
 
+    case UPDATE_REPLY_CACHE:
+      if (!payload.id || !payload.draft) {
+        return state;
+      }
+
+      if (!state.replyCache) {
+        state.replyCache = {};
+      }
+
+      const replyTime = new Date().getTime();
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)